.github/workflows/scripts/check-ispc.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
+cd build
+bin/check_isa
+bin/ispc --support-matrix
+make check-all
```

Comment:
Should we also replace those calls to make by `cmake --build . --target check-all ispc_benchmarks test` ?

Reply:
Sounds like a good idea.